RELEASE_PROCESS.md

```diff
@@ -17,6 +17,12 @@ Users with permissions in the `@spectrum-web-components` organization on NPM can
 
 The docs site will publish automatically if the `#publish` string is included in the commit message and the check suite runs successfully.
 
+In the case of a publishing error, `yarn lerna-publish` will no longer work. Instead, you'll need to run:
```

Comment:
If there is a failure _after_ tags have been published to GitHub?

Reply:
Possibly add a check like:

- visit https://github.com/adobe/spectrum-web-components
- see if "(your username) chore: release new versions" is the most-recent commit?

Reply:
That or check: https://github.com/adobe/spectrum-web-components/tags for recent versions, but either work for me.

Reply:
Ping for tomorrow. 🙏🏼

Reply:
@Westbrook how's that?